lib/collection/group.ts

```diff
@@ -11,17 +9,29 @@ export type Index = Array<PrimaryKey>;
 export type InstanceContext = (() => Collection) | (() => Pulse);
 
 export class Group<DataType = DefaultDataItem> extends State<Array<PrimaryKey>> {
-  _masterOutput: Array<DataType> = [];
-  missingPrimaryKeys: Array<PrimaryKey> = [];
+	_output: Array<DataType> = [];
+	_states: Array<State<DataType>> = []; // States of the Group
+	notFoundPrimaryKeys: Array<PrimaryKey> = [];
```

Comment:
not really necessary to change this

Reply:
sure.. for me notFound had sounded more suitable... but missing is also ok